test/functional/autocmd/bufenter_spec.lua

```diff
@@ -33,9 +33,10 @@ describe('autocmd BufEnter', function()
   end)
 
   it('triggered by ":split normal|:help|:bw"', function()
+    -- Add runtime from build dir for doc/tags (used with :help).
+    command(string.format([[set rtp+=%s/runtime]], helpers.test_build_dir))
     command("split normal")
     command("wincmd j")
-    command("helptags runtime/doc")
     command("help")
```

Comment:
Maybe we could have a `TestHelp` command instead that would adjust `rtp` then before calling `help`?
But it's also OK maybe having it done 5 times manually?

Reply:
a function would be useful, if there's some reason we can't just do it once in `clear()`.

Reply:
https://github.com/neovim/neovim/commit/384e519c78249fe59c1e3945828bd8d276d2ba3b did not work - i.e. that would need other test adjustments then.

Reply:
No, I mean just do whatever you're already doing, except in `clear()`.

Reply:
So the nearest `clear()` to the tests that need it, or in the default/global clear?  (the latter would be the same I guess with regard to unrelated/different failures then)

Reply:
I was thinking global clear, but sounds like that is too much trouble. It's better to explicit I guess, anyway (i.e. `helpers.add_builddir_to_rtp()`).

Reply:
Good idea, done.